src/modelfree/ppo_baseline.py

```diff
@@ -206,22 +247,45 @@ def setup_logger(out_dir="results", exp_name="test"):
     timestamp = datetime.datetime.now().strftime(ISO_TIMESTAMP)
     out_dir = osp.join(out_dir, '{} {}'.format(timestamp, exp_name))
     os.mkdir(out_dir)
-    logger.configure(dir=osp.join(out_dir, 'mon'))
+    logger.configure(dir=osp.join(out_dir, 'mon'),
+                     format_strs=['tensorboard', 'stdout'])
     return out_dir
 
 
 ppo_baseline_ex = Experiment("ppo_baseline")
 ppo_baseline_ex.observers.append(FileStorageObserver.create('my_runs'))
 
 
+@ppo_baseline_ex.named_config
```

Comment:
Can some of these be inherited from `default_ppo_config` and so do not need to be specified again?

Conversely, this defines some params (`env_wrapper_type`, ...) that do not appear in `default_ppo_config`. Should they defined in `default_ppo_config` too?

Reply:
Yes this has been taken care of.